socorro/unittest/external/es/base.py

```diff
@@ -774,6 +794,15 @@ def test_with_version(self):
     return decorated
 
 
+class SuperSearchWithFields(SuperSearch):
```

Comment:
I'm not convinced about this. Every new class comes with a cost of noise. I know this is just a part of tests, but it's part of the base module so it might get reused. 

It might be more of an advanced pattern but at least it's clear what it does when you view it and you don't have to go and look up the class definition:
```pyhon
self.supersearch = SuperSearchWithFields(config=self.config)
self.api = functools.partial(self.supersearch.get, _fields=self.supersearch.get_fields())
```